pkgs/tools/misc/pridecat/fix_install.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/Makefile b/Makefile
+index 815c27e..d4bc0b3 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,4 +1,5 @@
+ CXX ?= clang
++out ?= /usr/local
```

Comment:
This patch is a bit redundant because we can just set the out variable with makeFlags, and create the out directory in the preInstall phase

Reply:
the default make file has the install hard coded to `/usr/local/bin`, its pretty much just convince to set it to out

Reply:
Oh whoops, I misread your patch